src/AltinnCore/Common/Services/Interfaces/IGitea.cs

```diff
@@ -18,12 +18,12 @@ public interface IGitea
         Task<AltinnCore.RepositoryClient.Model.User> GetCurrentUser();
 
         /// <summary>
-        /// Create repository for th organisation
+        /// Create repository for the org.
```

Comment:
and here

Reply:
Have used org since that variable is explained to more detail in the parameter list.